src/actions/myStatsActions.js

```diff
@@ -0,0 +1,37 @@
+import { SET_USER_ADDRESS, SET_USER_LOADING, SET_USER_NEUMARKS } from './constants';
+import loadUserNeumarks from '../web3/loadUserNeumarks';
+
+export function setUserAddressAction(address) {
+  return {
+    type: SET_USER_ADDRESS,
+    payload: {
+      address,
+    },
+  };
+}
+
+export function setUserLoadingAction(loading) {
+  return {
+    type: SET_USER_LOADING,
+    payload: {
+      loading,
+    },
+  };
+}
+
+export function setUserNeuMarksAction(neumarkAmmount) {
+  return {
+    type: SET_USER_NEUMARKS,
+    payload: {
+      neumarkAmmount,
+    },
+  };
+}
+
+export const setUserAddress = address => async (dispatch) => {
+  dispatch(setUserAddressAction(address));
+  dispatch(setUserLoadingAction(true));
+  const neumarks = await loadUserNeumarks(address);
+  dispatch(setUserLoadingAction(false));
+  dispatch(setUserNeuMarksAction(neumarks));
```

Comment:
i think that `setUserNeuMarksAction` could set loading to false in reducer. So `dispatch(setUserLoadingAction(false));` is not really needed.